api/camunda/camunda-openapi.yaml

```
@@ -3790,6 +3859,123 @@ paths:
         "500":
           $ref: "#/components/responses/InternalServerError"
 
+  /batch-operations/{batchOperationKey}/cancel:
+    put:
+      tags:
+        - Batch operation
+      operationId: cancelBatchOperation
+      summary: Cancel Batch operation
```

```suggestion
      summary: Cancel batch operation
```

Unnecessary capitalization. I'll push an update upstream.